doc/deployment_guide.md

```diff
@@ -49,5422 +49,6234 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
+
+
+> Note: Please contact your Dell EMC sales representative for a detailed parts list or contact Dell EMC Professional Services team for OpenStack using the email [rhosp.ra.ps@emc.com](malito:rhosp.ra.ps@emc.com)
+
+### Version 13.2 features
+
+**The following features are added in the JetPack Automation and the Ready Architecture for RedHat OpenStack:**
+
+* Support for the lastest release of Red Hat OpenStack Platform 13.2 including the latest updates.
+* Support for latest release of RHEL 7.7 including the latest updates.
+* Support for Red Hat Ceph Storage version 3.2.2.
+* Added support for Jumbo Frames.
+* Added support for UEFI on all Overcloud nodes.
+* Added support for Dell EMC Networking S5248-ON switch.
+* Added support for 25GbE networking with Intel XXV710 network interface cards.
+* Added support for 100GbE networking with Mellanox ConnectX-5 network interface cards.
+* Added support for Red Hat Ceph Storage to use NVMe storage for OSD/Journal.
+* Enhancement of Huge Pages with JetPack CSP profile.
+* Enhancement of NUMA/CPU Pinning with JetPack CSP profile.
+* Added support of OVS-DPDK for 100G networking with Mellanox network interface cards (NICs).
+* Added support of SR-IOV offload for 100G networking with Connect X-5 Mellanox network interface cards (NICs) (Tech Preview in RedHat)
+* Added support of SR-IOV for 25GbE networking with Intel XXV710 network interface cards.
+* Added support for Cascade Lake 6230 processors
+* Added support for Distributed Virtual Router, (DVR).
+* Added support for auto-generation of .ini and .properties files for use in automated deployments.
+* Added support for VLAN aware VM.
+* Added support of Unity with Manila and Cinder
+* Added support of Octavia with Jetpack CSP profile
+* Added support of Barbican with Jetpack CSP profile
+* Added support of Satellite with Jetpack CSP profile
+* Added support for Bluestore
+
+
+## Before
+
+> Note: This guide assumes that you have racked the servers and networking hardware, and completed power and network cabling, as per the *Dell EMC Ready Architecture_for_Red_Hat OpenStack Platform Architecture Guide – Version 13.*
+
+The high-level steps required to install the Dell EMC Ready Architecture for Red Hat OpenStack Platform v13.2 using the automated installation procedures include:
+
+1.  Ensure that your environment meets the [Prerequisites](#Prerequisites)
+2.  Ensure that the [Dependencies](#Dependencies) are met.
+3.  *Determining Pool IDs*.
+4.  [Downloading and Extracting Automation Files]("https://github.com/dsp-jetpack/JetPack/").
+5.  [Preparing the Solution Admin Host Deployment](#Preparing-the-Solution-Admin-Host-Deployment).
```

Comment:
Should point to 13.2 branch that ll be available shortly : https://github.com/dsp-jetpack/JetPack/tree/JS-13.2

Reply:
Yes, I will update the link, when it will be available.